core/src/main/java/org/apache/calcite/adapter/enumerable/RexToLixTranslator.java

```diff
@@ -569,6 +569,18 @@ Expression translateCast(
     return scaleIntervalToNumber(sourceType, targetType, convert);
   }
 
+  private Expression handleNullUnboxingIfNecessary(
```

Comment:
Technically speaking this is not just `unboxing`.

`org.apache.calcite.adapter.enumerable.RexImpTable.NullAs#IS_NOT_NULL` means the expression `x` should be translated to `x is not null` kind of format.

In other words, `translate(x, NullAs.IS_NOT_NULL)` is equivalent to `translate(isNotNull(x), NOT_POSSIBLE)`, so `handleNull` sometimes adds `==null` or `!=null` kind of expressions.

Would you please rename accordingly?

Reply:
I totally agree. The way I am reading the method is handleNull, unboxing if necessary but if that creates confusion I will try to think of a better name.

Reply:
I was reading it as "handle NullUnboxing if necessary"

Reply:
What about the following name "handleNullAndUnboxIfNecessary"?

Reply:
I guess that would work.
`handleNull` would work as well.

Reply:
Done